src/components/report-viewer.tsx

```diff
@@ -130,7 +133,7 @@ export const ReportViewer = ({ report, onEditClick }: Props) => {
 
       {/* links to the author of the report */}
       <ProfileLink reporterId={reporterId} />
-      {onEditClick && <Button onClick={onEditClick}>Edit</Button>}
+      {onEditClick && <Button href={onEditClick}>Edit</Button>}
```

Comment:
Let's rename `onEditClick` to `reportEditorLink` since it is not an event handler anymore, but the name implies that it is.